site/en/docs/extensions/mv3/security/index.md

```diff
@@ -162,30 +167,19 @@ Avoid using `eval()` whenever possible to prevent attacks, as `eval()` will exec
 into it, which may be malicious.
 
 ```js
-var xhr = new XMLHttpRequest();
-xhr.open("GET", "https://api.example.com/data.json", true);
-xhr.onreadystatechange = function() {
-  if (xhr.readyState == 4) {
-    // WARNING! Might be evaluating an evil script!
-    var resp = eval("(" + xhr.responseText + ")");
-    ...
-  }
-}
-xhr.send();
+const fetchValue = await fetch("https://api.example.com/data.json");
+const jsonData = await fetchValue.json();
+// WARNING! Might be evaluating an evil script!
+const result = eval(jsonData);
```

Comment:
Do we even need this section? eval is no longer supported in MV3.

cc @jpmedley

Reply:
Even if it was still supported, why would evaluate JSON?

Reply:
Please remove as it doesn't make sense in MV3.